packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -479,6 +497,7 @@ export default {
       return {
         contentElement: this.$refs.content.$el,
         allowHTML: true,
+        appendTo: this.appendTo === 'body' ? this.anchor?.getRootNode()?.querySelector('body') : this.appendTo,
```

Comment:
Why is the default set to `this.anchor?.getRootNode()?.querySelector('body')` instead of `document.body`?

Reply:
it's for shadow dom reasons. When within a shadow dom we want it to get the shadow dom body, rather than the root body.